core-tests/src/test/java/org/mule/MuleEventTestCase.java

```diff
@@ -200,40 +197,32 @@ public void testFlowVarNamesRemoveMutable() throws Exception
     public void testFlowVarsNotShared() throws Exception
```

Comment:
It is, this is about if two events share the same flowVars map or not I believe.